great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -87,6 +90,23 @@ class ExpectColumnValuesToBeBetween(ColumnMapExpectation):
         [expect_column_value_lengths_to_be_between](https://greatexpectations.io/expectations/expect_column_value_lengths_to_be_between)
     """
 
+    min_value: Union[float, dict, datetime, None] = None
+    max_value: Union[float, dict, datetime, None] = None
+
+    _min_val = validator("min_value", allow_reuse=True)(validate_eval_parameter_dict)
+    _max_val = validator("max_value", allow_reuse=True)(validate_eval_parameter_dict)
+
+    @classmethod
+    @root_validator(pre=True)
+    def check_min_val_or_max_val(cls, values: dict) -> dict:
+        min_val = values.get("min_val")
+        max_val = values.get("max_val")
+
+        if min_val is None and max_val is None:
+            raise ValueError("min_value and max_value cannot both be None")
+
+        return values
```

Comment:
This specific Expectation requires one or the other (per `validate_configuration`)